Uyanda.Coffee.Application/Features/BeverageManagement/Requests/CustomerPurchaseCommand.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Uyanda.Coffee.Application.Features.BeverageManagement.Models;
+
+namespace Uyanda.Coffee.Application.Features.BeverageManagement.Requests
+{
+    public class CustomerPurchaseCommand
+    {
+        public InvoiceModel Invoice { get; set; }
+    }
```

Comment:
When you write code, you want it to represent the real world it models. When a user purchases coffee, they don't invoice the coffee company, the coffee company invoices the user. As such there should never be an intance where you send an _InvoiceModel_ to the backend.